docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md

```diff
@@ -94,14 +95,66 @@ class SalesMerchantPortalGuiDependencyProvider extends SprykerSalesMerchantPorta
 Make sure that the `ProductOfferMerchantOrderItemTableExpanderPlugin` is set up by opening `http://zed.mysprykershop.com/sales-merchant-portal-gui/orders`. Click on any of the orders and check that the *Merchant Reference* and *Product Offer SKU* are present.
 
 {% endinfo_block %}
+    
+#### Extend and validate PriceProducts with volume quantity for Offers
 
-### Add the Offer widget to MerchantDashobard
+Activate the following plugins:
```

Comment:
all this changes should be in Marketplace Product Offer Prices feature integration